src/apolo_app_types/helm/apps/fooocus.py

```diff
@@ -118,6 +123,30 @@ async def gen_extra_values(
                     ),
                 ]
             ),
+            health_checks=HealthCheckProbesConfig(
```

Comment:
nit, let's drop gen_extra_helm_args from those definitions, since it's redundant now